src/Layout/TopBar/index.tsx

```diff
@@ -32,7 +32,7 @@ const TopBar: React.FC = () => {
           {buttonText}
         </Button>
         <AccountSettingsBox>
-          <AccountSettings image={image} />
+          <AccountSettings image={user?.image!} />
```

Comment:
image jest opcjonalny? jeśli tak to albo nie renderujemy tego komponentu, albo w nim obsługujemy opcjonalny props